cloudops.go

```diff
@@ -48,6 +52,14 @@ type Compute interface {
 	// InspectInstanceGroupForInstance inspects the instance group to which the
 	// cloud instance with given ID belongs
 	InspectInstanceGroupForInstance(instanceID string) (*InstanceGroupInfo, error)
+	// SetInstanceGroupSize sets desired node count per availability zone
+	// for given instance group
+	SetInstanceGroupSize(instanceGroupInfo *InstanceGroupInfo,
```

Comment:
Pass instanceGroupID and location of instance group. Figure out ClusterName internally.

Reply:
Updated SetInstanceGroupSize() to only take instanceGroupID and location